code/game/atoms.dm

```diff
@@ -31,13 +31,6 @@
 
 /atom/Destroy()
 	set_opacity(0)
-
-
-	if(reagents)
-		qdel(reagents)
-		reagents = null
-
-	// Idea by ChuckTheSheep to make the object even more unreferencable.
 	invisibility = 101
 	return ..()

```

Comment:
Is the return ..() even necessary when nothing is above atoms? If so, what about the other atom/Destroy()?